src/ssr/reactProps.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * This is used to check if a property name is one of the React-specific
+ * properties and determine if that property should be forwarded
+ * to the React component
+ */
+
+/* Logic copied from ReactDOMUnknownPropertyHook */
+export default new Set([
```

Comment:
Set needs a polyfil, no? Maybe you could avoid using it here because its not necessarily.